AsyncDisplayKit/Layout/ASStaticLayoutSpec.mm

```diff
@@ -38,10 +39,8 @@ - (ASLayout *)measureWithSizeRange:(ASSizeRange)constrainedSize
 {
   CGSize maxConstrainedSize = CGSizeMake(constrainedSize.max.width, constrainedSize.max.height);
   
-  NSArray *children = self.children;
-  NSMutableArray *sublayouts = [NSMutableArray arrayWithCapacity:children.count];
-
-  for (id<ASLayoutable> child in children) {
+  NSMutableArray *sublayouts = [NSMutableArray array];
+  for (id<ASLayoutable> child in self) {
```

Comment:
I certainly can change, but `arrayWithCapacity:` does actually not have any effect at all, look at: https://www.objc.io/issues/7-foundation/collections/#should-i-use-arraywithcapacity